lib/jsdom/browser/resources/resource-loader.js

```diff
@@ -19,6 +19,20 @@ module.exports = class ResourceLoader {
     this._userAgent = userAgent;
   }
 
+  _readDataURL(urlRecord) {
+    const dataURL = dataURLFromRecord(urlRecord);
+    let timeoutId;
+    const promise = new Promise(resolve => {
+      timeoutId = setTimeout(resolve, 0, dataURL.body);
```

Comment:
The timing here makes it more consistent with other URL schemes and also makes it testable.